packages/apps-config/src/endpoints/production.ts

```diff
@@ -207,11 +207,10 @@ export const prodChains: EndpointOption[] = [
     }
   },
   {
-    info: 'nodle',
-    text: 'Nodle',
+    info: 'plasm',
+    text: 'Plasm',
```

Comment:
Why Plasm is replacing Nodle?

Reply:
We already added the nodle on top.
So we need to remove it on production.